modules/pageLoader.js

```diff
@@ -22,7 +22,7 @@ export default class PageLoader {
       if (event.key === 'Enter') {
         event.preventDefault();
         // console.log('pressed Enter');
```

Comment:
[OPTIONAL] Good job in trying to make sure the enter key is pressed before edits can happen, however, I think it will make the app much more difficult to use since you didn't give a user any form of documentation. A better approach would be to listen for a blur event or a change event